src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -744,6 +753,34 @@ private void AddExecutablePathCommand(IList<LaunchCommand> commands)
             commands.Add(new LaunchCommand("-file-exec-and-symbols " + exe, description, ignoreFailures: false, failureHandler: failureHandler));
         }
 
+        private void DetermineAndAddExecutablePathCommand(IList<LaunchCommand> commands)
+        {
+            // Runs a shell command to get the full path of the exe.
+            string absoluteExePath = string.Format(CultureInfo.InvariantCulture, @"shell readlink -f /proc/{0}/exe", _launchOptions.ProcessId);
```

Comment:
I've added a todo. Will need some testing with liblinux to see if it is more robust.

---
In reply to: [75936997](https://github.com/Microsoft/MIEngine/pull/422#discussion_r75936997) [](ancestors = 75936997,75935390)